var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -167,8 +176,21 @@ def configure(self):
 
     @when('@4')
     def configure(self):
+        if sys.platform == 'darwin':
+            # XXX(macos): Qt4 doesn't build with 10.10+ SDKs. Sorry.
+            sdkpath = which('xcrun')('--show-sdk-path', '--sdk', 'macosx10.9')
+            self.common_config_args.extend([
+                    '-sdk', sdkpath,
+                ])
+            if self.spec.compiler.name == 'clang' and \
+               str(self.spec.compiler.version).endwith('-apple'):
+                self.common_config_args.extend([
+                        '-platform', 'unsupported/macx-clang',
```

Comment:
since the old comment is now hidden, copy-paste for convenience of discussion:

`unsupported/macx-clang-libc++`? see https://github.com/Homebrew/homebrew-core/blob/master/Formula/qt.rb#L64